File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/facebook/FacebookSignInActivity.java
package com.jdroid.android.sample.ui.facebook;

import androidx.fragment.app.Fragment;

import com.jdroid.android.activity.FragmentContainerActivity;

public class FacebookSignInActivity extends FragmentContainerActivity {

	@Override
	protected Class<? extends Fragment> getFragmentClass() {
		return FacebookSignInFragment.class;
	}
}

File: jdroid-android-sample/src/main/java/com/jdroid/android/sample/ui/facebook/FacebookSignInActivity.kt
package com.jdroid.android.sample.ui.facebook

import androidx.fragment.app.Fragment

import com.jdroid.android.activity.FragmentContainerActivity

class FacebookSignInActivity : FragmentContainerActivity() {

    override fun getFragmentClass(): Class<out Fragment> {
        return FacebookSignInFragment::class.java
    }
}
